src/test/regress/expected/multi_metadata_sync_2.out

```diff
@@ -0,0 +1,1451 @@
+--
+-- MULTI_METADATA_SYNC
```

Comment:
Again the diff is pretty minor between this and `multi_metadata_sync_2.out`. 

To avoid adding this file: (a) Add ORDER BY to `SELECT unnest(master_metadata_snapshot()); ORDER BY 1`

(b) Add a normalization rule to get rid of ` CREATE INDEX mx_index ON public.mx_test_table USING btree (col_2) TABLESPACE pg_default`  the `TABLESPACE pg_default` part (which I guess you already do).

So, do we really need this?